### DISCUSSION TITLE
CSV Wide format date format issue

### DISCUSSION THREAD

**Discussion by dc9984** [CMSgov/hospital-price-transparency#87] [Q&A] [answered]
The wide version template shows the header date format requirement is YYYYMMDD. However the example file is in the format of MMDDYYYY and I also cannot get the CSV file to retain the YYYYMMDD format no matter what I try. Are there any known issues about this or any other suggestions? I cannot get passed the header portion of using the Online Validator because of this.

**Comment by knh762002** [comment]
Are you by chance opening the CSV file in excel and saving it before you use the validator?  I have noticed that Excel converts the date when 
 I have used it to open my CSVs.  Try opening the CSV file in a text editor after it is built to see if it the format is still correct before using Excel.  I hope that helps!

**Reply by dc9984** [reply]
Thank you!
I opened the file using Notepad and the date format is correct as I had saved it. However now today the validator is not tagging this as an error or warning.

**Reply by dc9984** [reply]
The validator is inconsistent with providing results. I corrected another header issue I had with the Code type and now it is back to saying the date format is wrong.

**Reply by dc9984** [reply, marked as answer]
Well I found an option on the web. I had to use the Custom format in the date cell and manually type in the format of YYY-MM-DD and it worked.
